### DISCUSSION TITLE
an equivalent to Vercel's Domains API ?

### DISCUSSION THREAD

**Discussion by vogdev** [coollabsio/coolify#2514] [Q&A]
hello is there an equivalent to Vercel's Domains API 
just to add update and remove domains in a project (nextjs)
if not how can i connect to the database that coolify saves the project domains in
![Screenshot 2024-06-12 at 17 47 51](https://github.com/coollabsio/coolify/assets/20149487/28a564a2-6a9d-47a4-af92-2f5046f17fd1)

**Comment by trm217** [comment]
I don't know the answer to that unfortunately.
However, a domain overview (per project and overall) is missing atm.
It's especially hard to keep track when having many apps deployed and domains assigned based on a wild-card A record.

**Comment by andrasbacsai** [comment]
I would add a view with all your domains set through Coolify, so you could have a central place to manage them, check their status (verified, unverified), if they are set properly in your DNS provider, something like this.

Is this what you looking for?

**Reply by trm217** [reply]
Yes, that's exactly what I was looking for 👍🏻
This would also allow for a better UI win domain inputs. (select domains + select prefix for wild-card domains)

**Reply by ofps** [reply]
@andrasbacsai If that also supports adding/removing a domain to/from a resource it's perfect 😌
